Sparkle/SUUpdateAlert.h

```diff
@@ -11,8 +11,9 @@
 
 #import "SUWindowController.h"
 #import "SUVersionDisplayProtocol.h"
+#import <WebKit/WebKit.h>
 
-@protocol SUUpdateAlertDelegate;
+@protocol SUUpdateAlertDelegate, WebPolicyDelegate, WebFrameLoadDelegate;
```

Comment:
I wasn't sure which SDK added them (I've started getting warning in 10.11), so they're for back-compat.